en/ui-system/components/editor/webview.md

```diff
@@ -0,0 +1,210 @@
+
+@ccclass('WebViewCtrl')
+export class WebViewCtrl extends Component {
+    @property(WebView)
+    webview = null;
+
+    start () {
+        // The Test here is a global function defined in your webView's internal page code
+        this.webview.evaluateJS('Test()');
+    }
+}
+```
+
+#### Note: Cross domain issues on Web platform need to be resolved by yourself
+
+### WebView internal pages call external code
+
+At present, the mechanism of Android and IOS is to determine whether the key of URL prefix is the same as that of the URL prefix by intercepting the jump, and then callback if the same.
+
+1. Setting the URL prefix keyword through `setJavascriptInterfaceScheme`
+2. The callback function is set by `setOnJSCallback`, and the function parameter is URL
+
+```ts
+import { _decorator, Component, WebView } from 'cc';
+const { ccclass, property } = _decorator;
+
+@ccclass('WebViewCtrl')
+export class WebViewCtrl extends Component {
+    @property(WebView)
+    webview = null;
+
+    // Setting in onLoad will make the callback useless, so we must set the cc.WebView callback in the start cycle.
+    start () {
+        // Here are the keywords that are agreed with the internal page
+        // Please set the scheme with lower case, the native won't identify the uppercase char scheme.
+        let scheme = "testkey";
+
+        function jsCallback (target, url) {
+            // The return value here is the URL value of the internal page, and it needs to parse the data it needs.
+            let str = url.replace(scheme + '://', ''); // str === 'a=1&b=2'
+            // webview target
+            console.log(target);
+        }
+
+        this.webview.setJavascriptInterfaceScheme(scheme);
+        this.webview.setOnJSCallback(jsCallback);
+    }
+}
+```
+
+So when you need to interact with WebView through an internal page, you should set the internal page URL: `testkey://(the data you want to callback to WebView later)`. WebView internal page code:
+
+```html
+<html>
+<body>
+    <dev>
+        <input type="button" value="Trigger" onclick="onClick()"/>
+    </dev>
+</body>
+<script>
+    function onClick () {
+        // One of them sets up the URL scheme
+        document.location = 'testkey://a=1&b=2';
+    }
+</script>
+</html>
+```
+
+Because of the limitations of Web platform, it can not be implemented by this mechanism, but internal pages can interact in the following ways:
+
+```html
+<html>
+<body>
+    <dev>
+        <input type="button" value="Trigger" onclick="onClick()"/>
+    </dev>
+</body>
+<script>
+    function onClick () {
+        // The parent here is actually the window of the WebView layer, 
+        // so that you can access the function defined in CC
+        parent.cc.TestCode();
+        // If TestCode is defined on window, then
+        parent.TestCode();
+    }
+</script>
+</html>
+```
+
+#### Stressed once: Cross domain issues on Web platform need to be resolved by yourself
```

Comment:
#### Stressed once: Cross domain issues on Web platform need to be resolved by yourself
-->
> **Note**: **Stressed once more** Cross-domain issues on Web platform need to be resolved by you, the developer.